starport/services/chain/faucet.go

```diff
@@ -90,6 +91,13 @@ func (c *Chain) Faucet(ctx context.Context) (cosmosfaucet.Faucet, error) {
 		faucetOptions = append(faucetOptions, cosmosfaucet.Coin(amount, amountMax, denom))
 	}
 
+	rateLimitWindow, err := time.ParseDuration(conf.Faucet.RateLimitWindow)
```

Comment:
Looking really good! 👏 

Let's check here if `conf.Faucet.RateLimitWindow != ""` otherwise, empty value always will cause `serve` command to fail.

Reply:
Right.
Sorry, I overlooked that. :-(

Reply:
How do I run all the tests before committing my changes?

Reply:
You can execute `script/test-integration`, but it takes plenty of time.

Reply:
Oh well, i can just run it overnight. It's better than making redundant commits and cluttering the history over stupid but critical mistakes.

Reply:
Not a problem at all! We squash the commits when merging to develop.